### ISSUE TITLE
java.lang.VerifyError: Bad type on operand stack with com/sun/net/httpserver/spi/HttpServerProvider

### ISSUE THREAD

**Issue by jasontatton** [Concurnas/Concurnas#10]
The following code with jdk 11 on windows compiles ok but fails with a java.lang.VerifyError

```
from com.sun.net.httpserver import HttpServer, HttpContext, HttpExchange, HttpHandler
import java.io.OutputStream
import java.net.InetSocketAddress
 
class MyHandler with HttpHandler {
  def handle(exchange HttpExchange) void {
    val response = "Hi There!"
    exchange.sendResponseHeaders(200, response.getBytes().length)
    val os = exchange.getResponseBody()
    os.write(response.getBytes())
    os.close()
  }
}
 
def main() void {
  val server = HttpServer.create(new InetSocketAddress("localhost", 8500), 0)
  server.createContext("/", new MyHandler())
  server.start()
}
```

upon runtime this throws a java.lang.VerifyError:

```
java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    com/sun/net/httpserver/spi/HttpServerProvider.<clinit>()V @7: putstatic
  Reason:
    Type uninitialized 0 (current frame, stack[0]) is not assignable to 'java/lang/Object'
  Current Frame:
    bci: @7
    flags: { }
    locals: { }
    stack: { uninitialized 0 }
  Bytecode:
    0000000: bb00 da59 0157 57b3 00df 01b3 0091 b1
        at jdk.httpserver/com.sun.net.httpserver.HttpServer.create(HttpServer.java:136)
        at http$Globals$.main(http.conc:16)
        at $ConcEXE.apply($ConcEXE.java)
        at $ConcEXE.apply($ConcEXE.java)
        at java.base/com.concurnas.bootstrap.runtime.cps.IsoCore._runExecute(IsoCore.java:25)
        at java.base/com.concurnas.bootstrap.runtime.cps.Iso.execute(Iso.java:48)
        at java.base/com.concurnas.bootstrap.runtime.cps.Worker.run(Worker.java:56)
```

Likely to be a problem relating to fiberization of the HttpServerProvider clinit method

**Comment by jasontatton** [comment]
For now workaround for this bug is to run the code on Concurnas via the JVM version 1.8.

The bug appears confined to versions of the JVM 9 and above (including 11 as reported here)

**Comment by jasontatton** [comment]
This bug is now resolved in Concurnas version 1.14.017 onwards.
